changelogs/fragments/504_postgresql_ext.yml

```diff
@@ -1,2 +1,2 @@
 minor_changes:
-- "postgresql_privs - added idempotence when version=latest (https://github.com/ansible-collections/community.postgresql/pull/504)."
+- "postgresql_ext - added idempotence when version=latest (https://github.com/ansible-collections/community.postgresql/pull/504)."
```

Comment:
is antsibull changelog ok with this being quoted like this @Andersson007 ? i've never seen it this way.

Reply:
@hunleyd yep, and it's sometimes necessary, e.g. when you have `:` in a sentence. The quotes will not appear in the changelog